x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/monitor_config/locations.tsx

```diff
@@ -65,18 +65,25 @@ export const ServiceLocations = ({
     <EuiFormRow label={LOCATIONS_LABEL} error={errorMessage} isInvalid={errorMessage !== null}>
       <EuiCheckboxGroup
         options={locations.map((location) => {
-          const badge =
+          let badge =
             location.status !== LocationStatus.GA ? (
               <EuiBadge color="warning">Tech Preview</EuiBadge>
```

Comment:
Not edited here, but looks like this needs i18n, as well as the other badges in this file.